resources/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 rm -rf dist/ && mkdir -p dist/ &&
-babel src --ignore __tests__ --out-dir dist/ &&
+babel src --ignore __tests__ --out-dir dist/ --stage=0 &&
```

Comment:
Can we adjust it to only add that one feature so it's more explicit? stage=0 adds a lot of random stuff.